docs/getting-started/my-account/README.md

```diff
@@ -29,7 +29,7 @@ To open your personal settings in OpenProject, click on your user icon in the to
 
 Choose **My account**.
 
-![my-account](my-account.png)
+![my-account-profil](Bildschirmfoto 2022-03-02 um 14.40.32.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.